snappy/xerial/xerial_encode_fuzz_test.go

```diff
@@ -0,0 +1,44 @@
+package xerial
```

Comment:
Please merge into `fuzz_test.go` - and TBH I don't really see where this adds value. Are you just removing the sub-tests?

Reply:
Yeah. Just replace the existing.